assets/stylesheets/_bootstrap-bookingsync.scss

```diff
@@ -65,6 +65,7 @@
 @import "bookingsync/form";
 @import "bookingsync/sheet";
 @import "bookingsync/switch";
+@import "bookingsync/bsy-switch";
```

Comment:
Looks very odd to have this prefix here, as well as the duplication.

Reply:
Will `ember-switch` make more sense, not sure how to deal duplication since switch is still used for Rails gem? I was thinking using separate files for Rails and Ember styles but not sure that makes sense...

Reply:
@JovicaSusa This is really about Ember Toggle here, so what about `bookingsync/ember-toggle` and loaded after `bookingsync/ember-power-select` then?